### PR TITLE
Documentation update to version 1.3

### PR DESCRIPTION
**Pull request by lxy2304**

### CODE REVIEW

File: docs/source/changelog.rst
Comment (msonderegger) [inline docs/source/changelog.rst:1]: There have been more changes than this, right?

File: docs/source/conf.py [hunk 1]
```diff
@@ -97,14 +97,9 @@
 copyright = '2015-2019, Montreal Corpus Tools'
```
Comment (msonderegger) [inline docs/source/conf.py:97]: should this now be 2015-2024?

**Review by msonderegger — COMMENTED** [review]
Looks good, I put a few little comments to consider. I tagged @mmcauliffe for review as well, but this looks fairly minor and if he doesn't have time it's fine.

File: docs/source/changelog.rst
Comment (lxy2304) [reply docs/source/changelog.rst:1]: Just updated with more changes that I consider notable. I think the more minor ones (like tutorials update, specifying dependencies versions etc.) should not go here?

**Review by msonderegger — APPROVED** [review]

**Pull request merged**